integration test region_merge often meets `query execution interruption` error

## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

   run integration test region_merge in CI

2. What did you expect to see?

   CI runs successfully

3. What did you see instead?

```

[2021-08-29T02:14:56.573Z] [Sun Aug 29 10:14:53 CST 2021] <<<<<< START cdc server in region_merge case >>>>>>
[2021-08-29T02:14:56.573Z] + cdc.test -test.coverprofile=/tmp/tidb_cdc_test/cov.region_merge.cli.1782.out cli changefeed create --sink-uri=mysql://normal:123456@127.0.0.1:3306/
[2021-08-29T02:14:56.573Z] Create changefeed successfully!
[2021-08-29T02:14:56.573Z] ID: b1bf790e-bb0e-42b4-bfd1-fc40248174d9
[2021-08-29T02:14:56.573Z] Info: {"sink-uri":"mysql://normal:123456@127.0.0.1:3306/","opts":{"_changefeed_id":"cli-verify"},"create-time":"2021-08-29T10:14:53.713574735+08:00","start-ts":427348012224217089,"target-ts":0,"admin-job-type":0,"sort-engine":"unified","sort-dir":"","config":{"case-sensitive":true,"enable-old-value":true,"force-replicate":false,"check-gc-safe-point":true,"filter":{"rules":["*.*"],"ignore-txn-start-ts":null},"mounter":{"worker-num":16},"sink":{"dispatchers":null,"protocol":"default"},"cyclic-replication":{"enable":false,"replica-id":0,"filter-replica-ids":null,"id-buckets":0,"sync-ddl":false},"scheduler":{"type":"table-number","polling-time":-1}},"state":"normal","history":null,"error":null,"sync-point-enabled":false,"sync-point-interval":600000000000,"creator-version":"v5.0.3-68-gcffb0fd1"}
[2021-08-29T02:14:56.573Z] PASS
[2021-08-29T02:14:56.573Z] coverage: 4.8% of statements in github.com/pingcap/ticdc/...
[2021-08-29T02:14:56.573Z] + set +x
[2021-08-29T02:14:56.573Z] split_and_random_merge scale: 100
[2021-08-29T02:15:03.124Z] *************************** 1. row ***************************
[2021-08-29T02:15:03.124Z] count(distinct region_id): 91
[2021-08-29T02:15:07.301Z] split_and_random_merge scale: 200
[2021-08-29T02:15:13.852Z] *************************** 1. row ***************************
[2021-08-29T02:15:13.853Z] count(distinct region_id): 143
[2021-08-29T02:15:18.062Z] split_and_random_merge scale: 400
[2021-08-29T02:15:36.174Z] ERROR 1317 (70100) at line 1: Query execution was interrupted
[2021-08-29T02:15:36.174Z] *************************** 1. row ***************************
[2021-08-29T02:15:36.174Z] count(distinct region_id): 372
[2021-08-29T02:15:48.358Z] ERROR 1317 (70100) at line 1: Query execution was interrupted
[2021-08-29T02:16:10.300Z] make: *** [integration_test_mysql] Error 1
```

4. Versions of the cluster

    - TiCDC version (execute `cdc version`):

        master@https://github.com/pingcap/ticdc/commit/b5ebeaf31fef9235ae3ef53ea663341f40160f58

--- a/tests/region_merge/run.sh
+++ b/tests/region_merge/run.sh
@@ -22,7 +22,7 @@ function split_and_random_merge() {
 }
 
 large_scale=(100 200 400 800 1600 3200 6400 12800 25600 51200)
-small_scale=(100 200 400)
+small_scale=(10 20 40)
 # in CI, we use the small data set
 test_scale=( "${small_scale[@]}" )
 

--- a/tests/region_merge/run.sh
+++ b/tests/region_merge/run.sh
@@ -22,7 +22,7 @@ function split_and_random_merge() {
 }
 
 large_scale=(100 200 400 800 1600 3200 6400 12800 25600 51200)
-small_scale=(10 20 40)
+small_scale=(20 40 80)
 # in CI, we use the small data set
 test_scale=( "${small_scale[@]}" )
 

--- a/tests/region_merge/run.sh
+++ b/tests/region_merge/run.sh
@@ -45,8 +45,8 @@ function run() {
       run_kafka_consumer $WORK_DIR "kafka://127.0.0.1:9092/$TOPIC_NAME?partition-num=4&version=${KAFKA_VERSION}"
     fi
 
-    # set max_execution_time to 15s, because split region could block even region has been split.
-    run_sql "SET @@global.MAX_EXECUTION_TIME = 15000;" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
+    # set max_execution_time to 30s, because split region could block even region has been split.
+    run_sql "SET @@global.MAX_EXECUTION_TIME = 30000;" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
     run_sql "CREATE DATABASE region_merge;" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
     run_sql "CREATE TABLE region_merge.t1 (id bigint primary key);" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 

